client/package.json

```diff
@@ -4,19 +4,23 @@
   "license": "MIT",
   "angular-cli": {},
   "scripts": {
-    "update-shared": "(cd ../shared && node build.js)",
     "start": "ng serve",
+    "build": "ng build --prod",
+    "build:staging": "ng build --env=staging --prod --sourcemaps false --progress false",
```

Comment:
Turning off `sourcemaps` here as we don't need those in production builds. Speeds up the build process.